prettier.config.mjs

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
I added this file to ensure that my editor picked up Prettier defaults - which this repository seems to follow - rather than my own person defaults. Let me know if you'd like me to remove it from this PR